glusterd2/commands/peers/addpeer.go

```diff
@@ -141,6 +141,7 @@ func createPeerAddResp(p *peer.Peer) *api.PeerAddResp {
 	return &api.PeerAddResp{
 		ID:              p.ID,
 		Name:            p.Name,
+		Online:          true,
```

Comment:
Shouldn't we display this in the cli as well?

Reply:
@atinmu showing this field in peer add response won't make any sense in my opinion.
let me know your thoughts on this.

Reply:
Then why is it different to be shown in the API response especially we know that online field will be anyways true. To me, uniformity on the response should be maintained as much as possible in both API & cli response.

Reply:
we are having one common structure for peer add, peer edit and get peer response.
we are making use of the online field in peer list and peer get cli command.